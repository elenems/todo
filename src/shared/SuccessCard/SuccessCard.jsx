import React from "react";
import Typography from "@material-ui/core/Typography";
import Box from "@material-ui/core/Box";
import { MdDone } from "react-icons/md";
export default function SuccessCard(props) {
  return (
    <Box
      style={{
        padding: "20px 16px",
        border: "1px solid",
        marginBottom: "12px"
      }}
      color="#4caf50"
      display="flex"
      alignItems="center"
      justifyContent="space-between"
      className="reveal card"
    >
      <Typography variant="h6" component="p">
        {props.message}
      </Typography>
      <div style={{ fontSize: "48px", lineHeight: "0em" }}>
        <MdDone color="#4caf50" />
      </div>
    </Box>
  );
}
