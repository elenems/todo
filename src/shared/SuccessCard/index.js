import SuccessCard from './SuccessCard.jsx';
export default SuccessCard;