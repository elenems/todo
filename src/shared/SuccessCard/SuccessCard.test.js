import React from 'react';
import {shallow} from 'enzyme';
import SuccessCard from '../SuccessCard';
describe('Alert card', ()=>{
 it('Renders', ()=>{
     const wrapper = shallow(<SuccessCard />).debug();
     expect(wrapper).toMatchSnapshot();
 })
})