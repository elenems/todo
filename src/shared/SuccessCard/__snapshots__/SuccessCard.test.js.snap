// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert card Renders 1`] = `
"<Styled(MuiBox) style={{...}} color=\\"#4caf50\\" display=\\"flex\\" alignItems=\\"center\\" justifyContent=\\"space-between\\" className=\\"reveal card\\">
  <WithStyles(ForwardRef(Typography)) variant=\\"h6\\" component=\\"p\\" />
  <div style={{...}}>
    <MdDone color=\\"#4caf50\\" />
  </div>
</Styled(MuiBox)>"
`;
